### ISSUE TITLE
Question about coordinate projection in TreeTransformerSparsePostv2

### ISSUE THREAD

**Issue by dreichCSL** [wenhuchen/Meta-Module-Network#10]
This layer in modular.py, TreeTransformerSparsePostv2 class:
self.coordinate_proj = nn.Linear(coordinate_dim, hidden_dim)

apparently expects 6 dimensional bounding box coordinate input and the pretrained model is trained as such. See also args.additional_dim in run_experiments.py which is set to 6 by default. Could you please explain why your bboxes have 6 dimensions or if I'm misinterpreting it? (I can't download your provided data to check if/what content is in dim 5&6.)
Thanks!

**Comment by dreichCSL** [comment]
Got it to work this way: 
run_experiments.py uses gqa_loader v2 by default (GQA_v2 in GQA.py), but GQA_v2 doesn't add those 2 coordinate dimensions. I copied the relevant code from GQA_v1, which creates these 2 extra dimensions (seems like width and height are added, see Lines 149 ff in GQA.py).
